ext/mysqli/mysqli.stub.php

```diff
@@ -0,0 +1,540 @@
+<?php
+
+class mysqli_driver
+{
+}
+
+class mysqli
+{
+    /** @return mysqli|null|false */
+    public function __construct(
+        ?string $host = null,
+        ?string $user = null,
+        ?string $password = null,
+        ?string $database = null,
+        ?int $port = null,
+        ?string $socket = null
+    );
+
+    /** @return bool|null */
+    public function autocommit(bool $mode);
+
+    /** @return bool|null */
+	public function begin_transaction(int $flags = 0, string $name = '');
```

Comment:
The default value of the `$name` parameter can't be `''` because of this code fragment:
```C
if (!name_len) {
	php_error_docref(NULL, E_WARNING, "Savepoint name cannot be empty");
	err = TRUE;
}
```
If the argument is not passed then the `name` and `name_len` variables remain `NULL` and `-1` respectively.

That said, the default value of the `$name` should be `UNKNOWN` instead.